zold.gemspec

```diff
@@ -68,6 +68,7 @@ and suggests a different architecture for digital wallet maintenance.'
   s.add_runtime_dependency 'xcop', '~>0.5'
   s.add_development_dependency 'codecov', '0.1.10'
   s.add_development_dependency 'minitest', '5.11.3'
+  s.add_development_dependency 'pry', '0.11.3'
```

Comment:
@liligga you are not using this gem. remove it.

Reply:
Ok I will. But actually this gem is used for debugging purposes. It would be stupid if I left any usage signs of it.